certora/specs/ConsistentState.spec

```diff
@@ -32,7 +36,7 @@ invariant timelockInRange()
     timelock() <= maxTimelock() && timelock() >= minTimelock()
 {
     preserved {
-        require isPendingTimelockInRange();
+        requireInvariant pendingTimelockInRange();
```

Comment:
It does the same thing (so it does not help the prover), but `requireInvariant` is safer than `require` because invariants are always verified. Whereas with `require` you always have to make sure that the formula that is passed (here it's `isPendingTimelockInRange()`) is true: if it's false then the verification will trivially pass, which makes the rule essentially useless